tests/framework/extensions/clusters/clusters.go

```diff
@@ -122,6 +123,63 @@ func CheckServiceAccountTokenSecret(client *rancher.Client, clusterName string)
 	}
 }
 
+// CreateRancherBaselinePSACT creates custom PSACT called rancher-baseline which sets each PSS to baseline.
+func CreateRancherBaselinePSACT(client *rancher.Client, psact string) error {
+	_, err := client.Steve.SteveType(PodSecurityAdmissionSteveResoureType).ByID(psact)
+	if err == nil {
+		return nil
+	}
+
+	template := &v3.PodSecurityAdmissionConfigurationTemplate{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: psact,
+		},
+		Description: "This is a custom baseline Pod Security Admission Configuration Template. " +
+			"It defines a minimally restrictive policy which prevents known privilege escalations. " +
+			"This policy contains namespace level exemptions for Rancher components.",
+		Configuration: v3.PodSecurityAdmissionConfigurationTemplateSpec{
+			Defaults: v3.PodSecurityAdmissionConfigurationTemplateDefaults{
+				Enforce: "baseline",
+				Audit:   "baseline",
+				Warn:    "baseline",
+			},
+			Exemptions: v3.PodSecurityAdmissionConfigurationTemplateExemptions{
+				Usernames:      []string{},
+				RuntimeClasses: []string{},
+				Namespaces: []string{
+					"ingress-nginx",
```

Comment:
should all this be hard-coded? Is there a reason a user would want to add a namespace to this list?

Reply:
For baseline, the namespaces have to be the same as `rancher-restricted`, we have that documented for users as well.